Ghidra/Features/Base/src/main/java/ghidra/program/util/VarnodeContext.java

```diff
@@ -838,10 +838,13 @@ public Address getLastSetLocation(Register reg, BigInteger bval) {
 				continue;
 			}
 			BigInteger rbval = rval.getUnsignedValue();
-			if (bval.equals(rbval)) {
+			if (bval != null && bval.equals(rbval)) {
```

Comment:
```suggestion
			if (bval == null || bval.equals(rbval)) {
```

845-847 could then be removed. Just a suggestion though.